spacetemplate/assets/agile.yaml

```diff
@@ -92,6 +92,28 @@ work_item_types:
         - No Plan to Implement
         - Resolved
         - Closed
+    "system.metastate":
+      label: Meta State
+      description: The meta-state of the work item
+      read_only: no
+      required: no
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        # This will allow other WITs to overwrite the values of the state.
+        rewritable_values: yes
+        # the sequence of the values need to match the sequence of the 
```

Comment:
Should it say this instead? *The __number__ of values here need to match the number of values in the `system.state` field definition.*

Reply:
Fixed.